### PR TITLE
Debugging tools

### PR DESCRIPTION
**Pull request by acaetano**
Tools used to explore objects returned by the Java engine and better plug Nashorn objects into it

### CODE REVIEW

**Comment by thc202** [comment]
- The script names should follow camel case.
- The changelog should be updated to mention the new script.
- It's preferable to rebase your local branches than merge.

**Comment by thc202** [comment]
@acaetano ping

**Comment by acaetano** [comment]
@thc202 Done.

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

**Comment by acaetano** [comment]
@kingthorin Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+### Added 
+- standalone/devTools.js - Tools used to explore objects returned by the Java engine and better plug Nashorn objects into it
```
Comment (thc202) [inline CHANGELOG.md:8]: Should be added to the existing section.

**Comment by acaetano** [comment]
@thc202 Done.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by kingthorin** [comment]
Thanks

**Review by kingthorin — APPROVED** [review]

**Comment by psiinon** [comment]
Love it - really useful script!

**Pull request merged**